content/docs/guides/converting-gatsby-to-tina.mdx

```diff
@@ -0,0 +1,464 @@
+    const value = createFilePath({ node, getNode })
+    createNodeField({
+      name: `tinaMarkdown`,
+      node,
+      value: JSON.stringify(
+        parseMDX(
+          node.body,
++          {
++           type: "rich-text",
++           name: "body",
++           label: "Body",
++           isBody: true,
++           },
++           imageCallback
+        )
+      ),
+    })
+// ...
+```
+
+Now that we've defined our new custom node we should be ready to use it in our graphql query. Open `src/templates/blog-post.js` and add our custom node to the page query.
+
+```
+export const pageQuery = graphql`
+  query BlogPostBySlug(
+    $id: String!
+    $previousPostId: String
+    $nextPostId: String
+  ) {
+    site {
+      siteMetadata {
+        title
+      }
+    }
+    mdx(id: { eq: $id }) {
+      id
++     fields {
++       tinaMarkdown
++     }
+// ...
+`
+```
+
+Now we're ready to parse the formatted markdown into our component. We'll also remove the `"children"` prop which normally contains the HTML from Gatsby's mdx parser.
+
+```diff
+const BlogPostTemplate = ({
+  data: { previous, next, site, mdx: post },
+  location,
+- children,
+}) => {
+  const siteTitle = site.siteMetadata?.title || `Title`
++ const tinaMarkdownContent = JSON.parse(post.fields.tinaMarkdown)
+  return (
+    <Layout location={location} title={siteTitle}>
+      <article
+        className="blog-post"
+        itemScope
+        itemType="http://schema.org/Article"
+      >
+        <header>
+          <h1 itemProp="headline">{post.frontmatter.title}</h1>
+          <p>{post.frontmatter.date}</p>
+        </header>
+-         {children}
++         <TinaMarkdown content={tinaMarkdownContent} />
+        <hr />
+  // ...
+  )
+}
+```
+
+Unfortunately, we'll need to move images to our media directory to have them appear on our pages. In this case there's only one image
+
+```powershell
+cp ./content/blog/hello-world/salty_egg.jpg static/images/salty_egg.jpg
+```
+
+We'll also need to update all of the lists to be the same format in `content/blog/hello-world/index.md`
+
+> You may need to update other elements on your own website. For a list of unsupported markdown elements for Tina see [our guide](https://tina.io/docs/reference/types/rich-text/?#unsupported-elements).
+
+````diff
+- - Red
++ * Red
+
+- - Green
++ * Green
+
+- - Blue
++ * Blue
+
+
+
+* Red
+
+* Green
+
+* Blue
+
+
+
+- - Red
++ * Red
+
+- - Green
++* Green
+
+- - Blue
++* Blue
+
+
+
+```markdown
+
++ * Red
+
+- - Green
++ * Green
+
+- - Blue
++ * Blue
+
+
+
+* Red
+
+* Green
+
+* Blue
+
+- - Red
++ * Red
+
+- - Green
++* Green
+
+- - Blue
++* Blue
+
+```
+````
+
+Now we should be able to read and edit our existing pages in Tina. We'll add some css to fix the images in our articles since they aren't being handled by to fix the width of our images since they're no longer being processed by gatsby. Add the following to the top of `src/style.css`. This will resize any images in our blog.
```

Comment:
```suggestion
Now we should be able to read and edit our existing pages in Tina. 

We'll add some css to fix the images in our articles since they aren't being handled by to fix the width of our images since they're no longer being processed by gatsby. 

Add the following to the top of `src/style.css`. This will resize any images in our blog.
```